lib/glimesh/graphql/schema/account_types.ex

```diff
@@ -41,10 +41,10 @@ defmodule Glimesh.Schema.AccountTypes do
       config(fn args, _ ->
         case Map.get(args, :streamer_id) do
           nil ->
-            {:ok, topic: [Glimesh.AccountFollows.get_subscribe_topic(:follows)]}
+            {:ok, topic: [get_subscribe_topic(:follows)]}
 
           streamer_id ->
-            {:ok, topic: [Glimesh.AccountFollows.get_subscribe_topic(:follows, streamer_id)]}
+            {:ok, topic: [get_subscribe_topic(:follows, streamer_id)]}
```

Comment:
```suggestion
            {:ok, topic: [AccountsFollows.get_subscribe_topic(:follows, streamer_id)]}
```

Reply:
@mja00 Is this a typo? Shouldn't `AccountsFollows` be `AccountFollows` - IE minus the 's'

Reply:
Yep! Edited the comment and fixed.